package.json

```diff
@@ -34,7 +34,9 @@
     "git-up": "^2.0.0"
   },
   "devDependencies": {
+    "browserify": "^13.0.1",
```

Comment:
Can you please remove `browserify` and `namy` as dev dependencies? Again, `babel-it` takes care of that.